lib/services/publish.test.js

```diff
@@ -237,4 +265,32 @@ describe(_.startCase(filename), function () {
         });
     });
   });
+
+  describe('_checkForArchived', function () {
+    const fn = lib[this.title];
+
+    it('rejects with an error object if the meta.archived property is true', function () {
+      const uri = 'http://example.com/uri@published';
+
+      db.getMeta.returns(Promise.resolve({archived: true}));
+
+      return fn(uri)
+        .catch(err => {
+          const expectedErr = {val:'', errors: {_checkForArchived: 'The page is archived and cannot be published'}};
```

Comment:
Please mind spacing when using objects. I know this is being picky 😛 but it helps for code style and readability
```suggestion
          const expectedErr = { val:'', errors: { _checkForArchived: 'The page is archived and cannot be published' } };
```